registrazione-sito-primario-pubblica-amministrazione.rst

```diff
@@ -0,0 +1,9 @@
+**Registrazione sito primario Pubblica amministrazione**
+========================================================
```

Comment:
```suggestion
========================================================
---------------------------------------------------------
```
https://docs.italia.it/italia/docs-italia/docs-italia-guide/it/bozza/appendice-2/titoli.html